packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockFLTCam.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+final class MockFLTCam: FLTCam {
```

Comment:
Can we have both MockFLTCam and FLTCam conform to the same protocol? (You don't have to do it in this PR).